.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,12 +1,13 @@
-## Remove this paragraph
+## Remove this section
 
-Please have a look at our developer documentation before submitting your Pull Request.
+Please have a look at [our developer documentation](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) before submitting your Pull Request.
 
-https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch
+Please note that the trac ticket, news fragment, and review submission portions of this process apply to *all* pull requests, no matter how small; if you don't do them, it's likely that nobody will even notice your PR needs a review.
 
 ## Contributor Checklist:
 
-* [ ] There is an associated ticket in Trac. Create a new one at https://twistedmatrix.com/trac/newticket
+* [ ] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/<!-- Create a new one at https://twistedmatrix.com/trac/newticket and replace this comment with the ticket number. -->
 * [ ] The changes pass minimal style checks (see: https://twistedmatrix.com/trac/wiki/TwistedDevelopment#GettingYourPatchAccepted )
 * [ ] I have created a newsfragment in src/twisted/newsfragments/ (see: https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles )
 * [ ] I have updated the automated tests.
+* [ ] I have submitted the associated Trac ticket for review by adding the word `review` to the keywords field in Trac, and putting a link to this PR in the comment; it shows up in https://twisted.reviews/ now.
```

Comment:
I feel like this checklist item is long, and could be broken into

* [ ] I have added a link to this PR to the Trac ticket
* [ ] I have submitted the associated Trac ticket (etc.)

Reply:
"the associated ticket in trac" part is the earlier step; this is long and wordy to ensure that it's super clear what is required to transition into the "review" state.

Which is not to say that there isn't room for improvement here, but making it multi-step isn't the help it needs.

Thanks so much for the review!  I'll land it now.